config/GLOBALS

```diff
@@ -58,24 +59,19 @@ if [ -f $thisdir/../../devconfig/GLOBALS ]; then
 else
 	# Set globals
 	if [ -f /net/mmx/mnt/app/eso/bin/apps/pc ]; then
-		TRAINVERSION="$(on -f mmx $PC s:30:1966084 2> /dev/null | $SED 's/[^a-zA-Z0-9_-]//g')"
-		MUVERSION="MU$(on -f mmx $PC s:30:1966083 2> /dev/null | $SED 's/[^a-zA-Z0-9_-]//g')"
-		FAZIT="$($PERSR 0 3221291024 2> /dev/null | $XXD -r -p 2> /dev/null | $SED 's/[^a-zA-Z0-9._-]//g')"
-		MNT="/net/mmx/mnt/boardbook/M.I.B-BACKUP"
-		NAND=$MNT/$MUVERSION/
+		MIBMO=1 # MHI2 only
 	else
```

Comment:
No user output about legacy support at this place?

We should at least write this to LOG file.

Reply:
no, we see it from train string